cmd/git-sync/main.go

```diff
@@ -359,6 +364,10 @@ func main() {
 		if *flExechookBackoff < time.Second {
 			handleError(log, true, "ERROR: --exechook-backoff must be at least 1s")
 		}
+
+		if *flOneTime {
+			exechookChannel = make(chan bool, 1)
```

Comment:
What if this was a field of Webhook and maaybe a method like "WaitForCompletion"  ?  Then this all gets localized even further.  Better or worse in your opinion?

Reply:
I like this localization suggestion, although I think I will implement this in the Hook**Runner** instead of Hook to minimize code changes, and because HookRunner already has the channel passed in.